tools/src/wyvern/tools/parsing/coreparser/WyvernParser.jj

```diff
@@ -264,6 +269,14 @@ AST ValDeclaration() :
   }
 }
 
+AST TypeAbbrevDeclaration() :
+{Token t = null; Type reference = null; Token alias = null; }
+{
+   t = <TYPE> alias = <IDENTIFIER> <EQUALS> reference = Type() <NEWLINE> {
+     return build.typeAbbrevDecl(alias.image, reference, loc(t));
+   }
+}
+
 AST VarDeclaration() :
```

Comment:
These changes to the parser, as well as the AST builder and the TypedAST, all look good!